packages/react-vapor/src/components/datePicker/DatePickerDropdown.tsx

```diff
@@ -160,15 +150,15 @@ export class DatePickerDropdown extends React.Component<IDatePickerDropdownProps
         return (
             <div className={classNames('date-picker-dropdown', this.props.className)}>
                 <div className={dropdownClasses} ref={(dropdown: HTMLDivElement) => (this.dropdown = dropdown)}>
-                    <span className={toggleClasses} onClick={() => this.handleClick()}>
-                        <span className="dropdown-selected-value">
+                    <span className={toggleClasses} onClick={() => !disabled && this.handleClick()}>

```

Comment:
You could add the `disabled` class in the `toggleClasses` I think it already does some magic for us, like graying everything. 

Also, have a look at [`pointer-events` rule in css](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events), setting this to none could help prevent the onClick prop to be triggered in the first place. Maybe we could add this rule to the `disabled` class in vapor if it isn't already there.